dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/AddRemoveKpis.ts

```diff
@@ -0,0 +1,113 @@
+import { EuTaxonomyDataForFinancials } from "@clients/backend";
+import { describeIf } from "@e2e/support/TestUtility";
+import { uploader_name, uploader_pw } from "@e2e/utils/Cypress";
+import {
+  companyViaApiAndEuTaxonomyDataForFinancialsViaForm,
+  fillEligibilityKpis,
+  fillField,
+} from "@e2e/utils/EuTaxonomyFinancialsUpload";
+import { FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { dateFormElement } from "@sharedUtils/components/DateFormElement";
+
+describeIf(
+  "As a user, I expect to be able to add and remove Eligible KPIs and send the form successfully",
+  {
+    executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+    dataEnvironments: ["fakeFixtures"],
+  },
+  function () {
+    beforeEach(() => {
+      cy.ensureLoggedIn(uploader_name, uploader_pw);
+    });
+
+    let testData: FixtureData<EuTaxonomyDataForFinancials>;
+
+    before(function () {
+      cy.fixture("CompanyInformationWithEuTaxonomyDataForFinancialsPreparedFixtures").then(function (jsonContent) {
+        const preparedFixtures = jsonContent as Array<FixtureData<EuTaxonomyDataForFinancials>>;
+        testData = getPreparedFixture("company-for-all-types", preparedFixtures);
+      });
+    });
+
+    it("Check", () => {
```

Comment:
Test name must be more verbose. Rather do something similar to the text in the `describeIf` function call's first argument.